rust/cloud-storage/authentication_service/src/api/user/post_get_names.rs

```diff
@@ -22,6 +22,7 @@ pub struct PostGetNamesRequestBody {
         post,
         path = "/user/get_names",
         operation_id = "get_user_names",
+        request_body = PostGetNamesRequestBody,
```

Comment:
utoipa infers this automatically from the extract::Json. no need to put here

it can also lead to confusion if we ever change the request body type and we may forget to update this


```suggestion
```